packages/toolpad-app/src/utils/uuid.ts

```diff
@@ -1,8 +1,10 @@
+import { webcrypto } from 'crypto';
+
 // credit: https://stackoverflow.com/a/2117523
 export function uuidv4() {
   // @ts-ignore
   return ([1e7] + -1e3 + -4e3 + -8e3 + -1e11).replace(/[018]/g, (c) =>
     // eslint-disable-next-line no-bitwise
-    (c ^ (crypto.getRandomValues(new Uint8Array(1))[0] & (15 >> (c / 4)))).toString(16),
```

Comment:
`crypto` seems to work as a global for me in node.js (`node -e 'console.log(!!crypto)'  `). Maybe you're on an outdated version?

Reply:
Adding https://github.com/mui/mui-toolpad/pull/1141

Reply:
Yeah I'm on Node 16, maybe the issue isn't there in 18.
I can start using 18, should be fine.